### PR TITLE
hover layout, consecutive search issue

### PR DESCRIPTION
**Pull request by MemboLembo**
- Consecutive search issue by deleting previous result.

- Added new child for anime-content__item to fix hover layout positioning.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ document.addEventListener("DOMContentLoaded", function (event) {
         e.preventDefault();
         const searchVar = userSearch.value;
         console.log(searchVar);
+        
+        const deleteItems = document.querySelectorAll('.search-result__anime-content__item');
+        deleteItems.forEach(item => {
+            item.remove();
+        });
```
Comment (Badrpas) [inline js/index.js]: Why do we need to perform the querySelector when we already have the `animeContent` element?

Comment (Badrpas) [reply js/index.js]: Can't we just clear it's contents?
```javascript
animeContent.innerHTML = '';
```

Comment (MemboLembo) [reply js/index.js]: Oh, yeah. Thats much better!

Comment (MemboLembo) [reply js/index.js]: I fixed it in ae47074.

**Pull request merged**